commands/command.gd

```diff
@@ -317,6 +323,9 @@ func _can_be_selected() -> bool:
 func _get_description() -> String:
 	return ""
 
+func _get_color() -> Color:
+	return Color()
```

Comment:
I think we should use `.get_theme_color("font_color", "Editor")`

Reply:
Collections are not a node but a resource so get_window().get_theme_color is unavailable